lists/cn/cn-usci.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": {
+    "en": "Unified Social Credit Identifier",
+    "local": "统一社会信用代码"
+  },
+  "url": "http://ngo.mps.gov.cn/ngo/portal/toInfogs.do",
+  "description": {
+    "en": "Unified Social Credit Identifiers (USCIs) are used for businesses and other entities in China as a general tax identification number (TIN), but the online registry mentioned here only shows non-domestic NGOs. There is also a website for businesses (http://gsxt.gov.cn) but you will need to know the organisation’s name in Chinese to find the USCI. NGOs and businesses should be able to provide their USCI easily. NGOs have to have their registration checked and reconfirmed every year in accordance with the Overseas NGO Management Law which came into effect in January 2017."
+  },
+  "coverage": [
+    "CN"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "charity",
+    "company"
+  ],
+  "sector": [],
+  "code": "CN-USCI",
+  "confirmed": true,
+  "deprecated": false,
+  "listType": "primary",
+  "access": {
+   "availableOnline": true,
+    "onlineAccessDetails": null,
+    "publicDatabase": "http://www.chinanpo.gov.cn/search/orgindex.html",
+    "guidanceOnLocatingIds": "This list only shows non-domestic NGOs. Your browser may be able to translate the page if necessary. First field in the form is the name of the organisation. Click the blue button and a new table with results should appear, with the Unified Social Credit Identifier in the third column.\nFor other domestic entities, ask the organisation for their USCI or go to http://gsxt.gov.cn. You will need the name of the organisation in Chinese.",
+    "exampleIdentifiers": "G1610000713522976L, G1330000MCW06863XQ, G11100007178352756",
+    "languages": ["zh"]
+  },
+  "data": {
+    "availability": [],
+    "dataAccessDetails": "",
+    "features": [],
+    "licenseStatus": "open_license",
```

Comment:
I can't see where an open license is granted. An [example page](https://ngo.mps.gov.cn/ngo/portal/viewInfogsNew.do?p_creditcode=G1610000713522976L&p_createdate=2017-06-30&p_tab=B&p_topmenu=3) includes a Copyright statement and the [Site Statement](https://ngo.mps.gov.cn/ngo/portal/viewJsp.do?p_jsp=2&p_topmenu=1) states that 

> The content of this website is provided by the Ministry of Public Security and the Overseas NGO Management Office of Provincial Public Security Organs. Any media, Internet sites and commercial organizations are not allowed to use the content published on this website for commercial original reprinting, nor can they distort or tamper with the content published on this website. The copyright involved in this website belongs to this website.
...
Any media or Internet site shall not reprint the information and service content provided by other units on this website without authorization. If reprinting is required, it must directly contact the corresponding provider to obtain legal authorization.

Reply:
Will change to "closed_license".